Package.swift

```diff
@@ -257,5 +257,18 @@ let package = Package(
       ],
       path: "Sources/Examples/RouteGuide/Server"
     ),
+
+    // Client for the PacketCapture example
+    .target(
+      name: "PacketCapture",
```

Comment:
shouldn't all targets be `GRPC` prefixed? Targets need to be unique across the whole binary (so all dependencies' targets must be globally unique)

Reply:
Isn't that only necessary for targets which are also products?

Reply:
@glbrntt no, there can only be _one module_ of a given name in the whole binary. Some people call it the `Utilities` module problem.

Reply:
Sure, but we have target specific dependencies right? So this module won't _be_ in the product, because it's an executable target.

Reply:
And I think in your particular case it will appear to work. But I think there's some weird issue when you do `swift package edit` where then even modules that aren't officially exported at all start to clash.

Reply:
Put another way, it's a leaf.

Reply:
@Lukasa I _think_ it's still an issue with `swift package edit` but let me try to create a repo.

Reply:
there we go: https://github.com/apple/swift-nio-ssl/commit/f9ac0c3486f8e9333e791caf135e3fe067e0e66c

Reply:
So I think the repro is: Create a package `C` that depends on `A` and `B`. Make `A` and `B` both have a target called `Target`. After that, in `C`, do `swift package edit A && swift package edit B`. And I think now you'll get the error :)

Reply:
Well that's stupid. Is there a Swift bug report for this?

Reply:
@Lukasa I don't think I filed one but this is a long time ago.

Reply:
@weissi I can't repro that. Plus, I thought the reasoning for the package description API in 5.2 where packages are explicitly named was to avoid this problem?

Reply:
@glbrntt that _is_ possible. Maybe it's just a problem in < 5.2? Tbh I don't fully understand what's going on but it used to break the release tools. But we do test with 5.0 there...

Reply:
Yeah, I can totally believe that it didn't work before 5.2. Any objections to leaving the naming as is @weissi?

Reply:
@glbrntt go for it. We can fix that at any time if it causes trouble.